src/Main.java

```diff
@@ -1,5 +1,91 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        int salary = 15000;
+        int total=0;
+        int i=0;
+        while (total<2_459_000)
+        {
+        i=i+1;
+        total=total+salary;
+            System.out.println("Месяц "+i+" сумма накоплений равна "+total+" рублей" );
+        }
+        int g=0;
+
+        while(g<10)
+        {
+            g=g+1;
+            System.out.print( " "+g+" " );
+        }
+        for (int t=10;t>0;t--){
+            System.out.print(" "+t+" ");
+        }
+        int y =12_000_000;
+        int people =12_000_000/1000;
+        int born=17;
+        int dead=8;
+
+        for(int t=1;t<=10;t++)
+        {
+            y=y+(people*(born -dead));
```

Comment:
```suggestion
            y = y + (y * (born - dead) / 1000);
```
надо умножать на текущее кол-во людей, т.к. оно меняется каждый год и смертность с рождаемостью каждый год разные в численном отношении